backend/src/Service/OrderLogs/OrderLogsService.php

```diff
@@ -27,16 +31,17 @@ public function createOrderLogs(OrderLogsCreateRequest $request): OrderLogsRespo
        return $this->autoMapping->map(OrderLogsEntity::class, OrderLogsResponse::class, $orderLog);
     }
     
-    public function createOrderLogsRequest(OrderEntity $order): OrderLogsResponse
+    public function createOrderLogsRequest(OrderEntity $order, StoreOwnerBranchEntity $branch = null): OrderLogsResponse
```

Comment:
Why branch argument may be null while when we create order there will be for sure a branch linked to it?